app/views/posts/show.html.erb

```diff
@@ -1 +1,20 @@
-<h1>Here your post with post_id and user_id</h1>
\ No newline at end of file
+<head>
+  <link rel="stylesheet" href="<%= asset_path('application.css') %>">
+</head>
+<div class="post">
+ <h2> <%= @post.title  %> #<%= @post.id  %> by <%= @post.author.name  %> </h2>
+  <p>
+  Comments:  <%= @post.comments_counter %> <br> Likes:  <%= @post.likes_counter %>
+   </p>
+     <p>
+  <%= @post.text %> 
+   </p>
+   </div>
+   <div class="comment">
+   <% @post.comments.each do |comment| %>
+   <div class="comment-container">
+  <p>Commenter-<%= comment.author.name %>-</p>
+  <p>comment number: <%= comment.id %> </p>
+  </div>
+<% end %>
```

Comment:
| Your Post Show | Wireframe Post Show |
| --- | --- |
| ![image](https://github.com/otmanTR/BlogAppRoR/assets/93039359/b9036a50-95e4-44ab-adb9-fdfbf646eedf) | ![image](https://github.com/otmanTR/BlogAppRoR/assets/93039359/af51593a-4722-4c1b-a52f-0353d65a8daa) |

- [ ] It would be nice if you make the image size and the content size similar as you can see in the wireframe to make your project follow the requirements :+1: but now your image size is a lot bigger than the expected size :fearful:
- [ ] It would be nice if you make the comments and likes counter at the right of the posts to match the wireframe :+1:
- [ ] It would be nice if you make show the comments instead of Comment 1 and Comment 2 because these are just placeholders in the wireframe to tell you that here you need to show the comments :smile: